src/Compiler/CompilerOptions.cpp

```diff
@@ -150,6 +177,14 @@ llvm::cl::opt<bool> onnxOpTransformReport("onnx-op-transform-report",
     llvm::cl::desc("Report diagnostic info for op transform passes."),
     llvm::cl::init(false), llvm::cl::cat(OMPassOptions));
 
+// Configuration states associated with certain options.
+// For example, when maccel is specified, NNPA can register
+// dependent libdnn.
+// This is just a simple string to vector map currently.
+// If it gets more complicated in the future, it can be
+// replaced by a class of its own.
+std::map<std::string, std::vector<std::string>> CompilerConfigMap;
```

Comment:
Wondering whether we could use LLVM containers here like SmallVector and StringRef.

Reply:
ok